### ISSUE TITLE
pairwise T-test always set to FALSE in `aldex()` function

### ISSUE THREAD

**Issue by Yixuan39** [ggloor/ALDEx_bioc#75]
Hello ALDEx2 Develop Team,

We @benjjneb found that the `paired.test` option is always set as FALSE in `aldex.ttest` no matter how it change in the `aldex` function. Can you check about it?

https://github.com/ggloor/ALDEx_bioc/blob/eb5a203850387bccbd6250e80954f6317e2e78e1/R/aldex.r#L108

Thank you

**Comment by boopalaarul** [comment]
I don't think this is setting paired.test as always FALSE. This is how R specifies default values for parameters.

**Comment by Yixuan39** [comment]
Shouldn't it be x.tt <- aldex.ttest(x, paired.test=paired.test, hist.plot=hist.plot, verbose=verbose) ?
These are arguments set by the `aldex` function
https://github.com/ggloor/ALDEx_bioc/blob/eb5a203850387bccbd6250e80954f6317e2e78e1/R/aldex.r#L94

**Comment by boopalaarul** [comment]
Yeah, in that case you're right. The subroutine should take parameters from the calling function.
